commit 0a9bb5da869e74f9929a0c72116c52f2e3a370d4 (refs/changes/62/262/1)
Author: Todd Whiteman <todd.whiteman@joyent.com>
Date:   2016-08-09T15:41:34-07:00 (3 years, 2 months ago)
    
    DOCKER-908: remove test files from npm install of sdc-docker-build

diff --git a/package.json b/package.json
index 91861e9..e4566f2 100644
--- a/package.json
+++ b/package.json
@@ -24,7 +24,7 @@
     "restify": "4.0.0",
     "rimraf": "2.4.3",
     "sdc-clients": "~9.1",
-    "sdc-docker-build": "git+https://github.com/joyent/sdc-docker-build.git#2c5cafc2275587ab6b3f714cce3f6f38cfe9a1a9",
+    "sdc-docker-build": "git+https://github.com/joyent/sdc-docker-build.git#91ac3b0ac9f9579aeb88f9f9d0a2ff9814e3cbe8",
     "semver": "2.2.1",
     "shell-escape": "^0.2.0",
     "sprintf": "0.1.1",
